global/nixos-install-oneclick/flake.nix

```diff
@@ -0,0 +1,35 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    disko.url = "github:nix-community/disko";
+    disko.inputs.nixpkgs.follows = "nixpkgs";
+  };
+  outputs = { self, nixpkgs, disko, ... }: {
+    # A bare NixOS configuration that can be ssh'ed to as root, with known password
+    # For testing/demo purposes only.
+    nixosConfigurations.oneclick = nixpkgs.lib.nixosSystem {
+      # TODO: can we parametrize this? Maybe via --override-input?
+      system = "aarch64-linux";
+      modules = [
+        ./disk-config.nix
+        disko.nixosModules.disko
+        ({ pkgs, ... }: {
+          environment.systemPackages = [ pkgs.htop ];
+          boot.loader = {
+            systemd-boot.enable = true;
+            efi.canTouchEfiVariables = true;
```

Comment:
Given [this warning](https://nixos.zulipchat.com/#narrow/stream/413948-nixos/topic/disko-install/near/429653019), we should probably talk a bit of boot configuration in this post.